src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/CommentViewModel.cs

```diff
@@ -10,5 +10,7 @@ public class CommentViewModel
         public DateTime Created { get; set; }
         public User CreatedBy { get; set; }
         public DateTime? Updated { get; set; }
+        public DateTime? Resolved { get; set; }
+        public User ResolvedBy { get; set; }
```

Comment:
At some point we should switch to using UserViewModel to avoid accidentally leaking any info about users to the frontend

Reply:
As discussed on standup, I'm leaving this as it is for now until we decide whether to split the resolved functionality into a separate endpoint.